lesson_7_exercises/ex1.dkc.yml

```diff
@@ -1,3 +1,34 @@
-sevices:
-   your_service:
+version: "3.8"
 
+x-labels: &default-info
+  labels:
+    maintainer: "J.L."
+    author: "J.L."
+    com.imatia.project: "docker-course"
+  environment:
+    LESSON_NUMBER: 7
+    EXERCISE_NUMBER: 1
+
+services:
+  ex1-nginx-base:
+    image: &nginx nginx:1.21.1
+    <<: *default-info
+    ports:
+      - "8011:80"
+    healthcheck:
+      test: ["CMD-SHELL", "curl --fail http://localhost:80/ || exit 1"]
+      interval: 15s
+      timeout: 3s
+
+  ex1-nginx-modified:
+    image: *nginx
+    depends_on:
+      - ex1-nginx-base
+          # condition: service_healthy # Specifying conditiions is not allowed in docker-compose 3 specification.
```

Comment:
creo que isto volve a estar dispoñible nas versións máis novas (v2 e v3 mesturadas e `docker compose` sen guión)